.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "\U0001F680 Feature Request"
```

Comment:
can you also conver this to a yaml style template so that we're being consistent across the repo for our templates 🙏🏻

Reply:
sorted out here 4ecfdb1